index.php

```diff
@@ -20,4 +20,4 @@
 
 error_reporting(E_ERROR);
```

Comment:
This is useful, another thing that might be handy to see errors is to revert these two options:
https://github.com/pkp/ojs/blob/main/config.TEMPLATE.inc.php#L502-L505

Reply:
That's a good idea. I'll add a note about that to the documentation.